src/main/windows/windowManager.js

```diff
@@ -360,3 +362,12 @@ export function restore() {
   const focused = BrowserWindow.getFocusedWindow();
   focused.restore();
 }
+
+export function handleHistory(event, offset) {
+  if (status.viewManager) {
+    const activeView = status.viewManager.getCurrentView();
+    if (activeView && activeView.view.webContents.canGoToOffset(offset)) {
+      activeView.view.webContents.goToOffset(offset);
```

Comment:
I'd add a try/catch around this line, on some nested redirections it might cause trouble.

In case of failure we should log the error, try to go to the server's url and turn off the bar.